--- a/fs/proc/version.c
+++ b/fs/proc/version.c
@@ -10,8 +10,8 @@
 static int version_proc_show(struct seq_file *m, void *v)
 {
 	seq_printf(m, linux_proc_banner,
 		utsname()->sysname,
-		utsname()->release,
+		"6.1.118-yuukari-ne-perf+-manmbanbu-perf+",
 		utsname()->version);
 	return 0;
 }
--- a/kernel/sys.c
+++ b/kernel/sys.c
@@ -1306,4 +1306,10 @@
 	memcpy(&tmp, utsname(), sizeof(tmp));
 	up_read(&uts_sem);
+
+	if (from_kuid_munged(current_user_ns(), current_uid()) != 0) {
+		strlcpy(tmp.release, "6.1.118-yuukari-ne-perf+-manmbanbu-perf+", sizeof(tmp.release));
+	}
+
 	if (copy_to_user(name, &tmp, sizeof(tmp)))
 		return -EFAULT;
